src/routes/+layout.svelte

```diff
@@ -1,6 +1,10 @@
 <script lang="ts">
+  import Header from "$lib/components/Header.svelte";
+  import Footer from "$lib/components/Footer.svelte";
   import "../app.postcss";
   import "../app.scss";
 </script>
 
+<Header />
 <slot />
```

Comment:
To make out HTML better an more descriptive, I always wrap this `<slot />` in a `<main>` tag.

Docs of the main tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main